ethcore/src/ethereum/ethash.rs

```diff
@@ -453,11 +453,14 @@ impl Ethash {
 		let parent_has_uncles = parent.uncles_hash() != &sha3::SHA3_EMPTY_LIST_RLP;
 
 		let min_difficulty = self.ethash_params.minimum_difficulty;
+
 		let difficulty_hardfork = header.number() >= self.ethash_params.difficulty_hardfork_transition;
-		let difficulty_bound_divisor = match difficulty_hardfork {
-			true => self.ethash_params.difficulty_hardfork_bound_divisor,
-			false => self.ethash_params.difficulty_bound_divisor,
+		let difficulty_bound_divisor = if difficulty_hardfork {
+			self.ethash_params.difficulty_hardfork_bound_divisor
+		} else {
```

Comment:
`true =>` 94 cases, `false =>` 95 cases. some of those will be the same `= if` fragment, but still, it's a lot more than 2 or 3. general convention is to use the `match` syntax since it's more compact. it's not a desperately hard rule, but generally preferable unless there are mitigating circumstances.